modules/serverless-event-delivery-admin-config.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * /serverless/event_workflows/serverless-event-delivery.adoc
+
+[id="serverless-event-delivery-admin-config_{context}"]
+= Configuring global event delivery failure parameters by using config maps
+
+Cluster administrators can configure global event delivery parameters for subscriptions by modifying the `config-br-delivery` config map.
+
+.Example config map
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-delivery
+  namespace: cloud-run-events
+data:
+  default-br-delivery-config: |
+    clusterDefaults:
+      deadLetterSink: <1>
+        ref:
+          apiVersion: serving.knative.dev/v1
+          kind: Service
+          name: <sink_name>
+      backoffDelay: <duration> <2>
+      backoffPolicy: <policy_type> <3>
+      retry: <integer> <4>
+    namespaceDefaults:
+      customized-ns: <namespace>
+        deadLetterSink: <1>
+          ref:
+            apiVersion: serving.knative.dev/v1
+            kind: Service
+            name: <sink_name>
+        backoffDelay: <duration> <2>
+        backoffPolicy: <policy_type> <3>
+        retry: <integer> <4>
+----
+<1> Configuration settings to enable using a dead letter sink. This tells the subscription what happens to events that cannot be delivered to the subscriber.
++
+When this is configured, events that failed to be consumed are sent to the dead letter sink destination. The event is dropped, no re-delivery of the event is attempted, and an error is logged in the system. In this example, the sink is a Knative service.
```

Comment:
> The event is dropped, no re-delivery of the event is attempted, and an error is logged in the system. In this example, the sink is a Knative service.

This is confusing. The event is not dropped, is sent to the dead letter sink.

> re-delivery of the event is attempted

Sending to the dead letter sink happens only after trying to retry to sending to the "original" destination, given the retry is configured. e.g. assuming retry is configured to 3 and dead letter sink is configured, the dispatch works as follows:

1. Try to send to subscription destination, fails
2. Try to send to subscription destination, fails
3. Try to send to subscription destination, fails
4. Try to send to dead letter sink

Reply:
Removed this module since there's no global (admin) config